pkgs/tools/filesystems/fatresize/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, parted, substituteAll, utillinux, ... }:
```

Comment:
The `...` can be dropped (any extra parameters can't be accessed within the expression anyway).